lib/client/weblogin.go

```diff
@@ -364,6 +385,78 @@ func SSHAgentLogin(ctx context.Context, login SSHLoginDirect) (*auth.SSHLoginRes
 	return out, nil
 }
 
+// SSHAgentPwdlessLogin requests a passwordless MFA challenge via the proxy.
+// If the credentials are valid, the proxy will return a challenge. We then
+// prompt the user to touch a device and if user wasn't explicitly defined,
+// also prompts the user to select a login name. If the authentication succeeds,
+// we will get a temporary certificate back.
+func SSHAgentPwdlessLogin(ctx context.Context, login SSHLoginPasswordless) (*auth.SSHLoginResponse, error) {
```

Comment:
Do you think it's worth testing this method explicitly? Maybe reusing the tc.Login test setup?

Reply:
I am having trouble running the test that would help me test this, so I will do this last. i honestly don't think there's anything wrong with the test, it's something I screwed up and I can't figure out what I did (it works for two folks with linux env). I do have a mac that I need to finish setting up so I guess this is a sign for me to switch over

Reply:
Let me know if you intend to follow up on this if it's solved.